postgres-demo/README.md

```diff
@@ -182,8 +182,9 @@ how many login attempts have occured for a particular account in the last
 five minutes,  for example to send an alert in case of a large
 number of failed attempts:
 
+
 ```postgresql
->> psql
+>> psql -h localhost -U postgres
```

Comment:
You could try this command to use psql through Docker:
```
docker exec -it apiary-postgres psql -h localhost -U postgres
```
If it works, let's update the command here?